Gemfile.lock

```diff
@@ -31,7 +31,7 @@ GEM
       public_suffix (>= 2.0.2, < 5.0)
     ast (2.4.2)
     builder (3.2.4)
-    concurrent-ruby (1.1.6)
+    concurrent-ruby (1.2.2)
```

Comment:
I ran ` bundle update app_profiler` so I am guessing it updates the dependencies too.

Reply:
looks like the minitest has some breaking changes. I opted to not update the dependencies at this time, just ran `bundle update --source app_profiler`